describe_test.go

```diff
@@ -48,7 +48,7 @@ MTI...........................: 0100
 Bitmap........................: 400000000000000000000000000000000000000000000000
 Bitmap bits...................: 01000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
 F000 Message Type Indicator...: 0100
-F002 Primary Account Number...: 4242424242424242
+F002 Primary Account Number...: 4****4242
```

Comment:
I think the output should match the original length. PANs are pretty known lengths, so we aren't revealing much by showing the actual length.